From 0ddf28ece68c104a2a9370667cc370e7c93400bf Mon Sep 17 00:00:00 2001
From: "user.email" <sunkar2710@gmail.com>
Date: Fri, 18 Aug 2023 12:11:38 +0600
Subject: [PATCH] Changed 2 lines in firstFile.txt in git_1

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index 00935f1..0e87b22 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-Line 1
-Line 2
+Line I
+Line II
 Line 3
 Line 4
 Line 5
-- 
2.41.0.windows.3

